src/encoded/schemas/mixins.json

```diff
@@ -99,13 +99,37 @@
     },
     "form_status": {
         "form_status": {
-            "title": "Form Status",
+            "title": "Complete?",
             "type": "string",
-            "default": "inverified",
-            "enum" : [
+            "default": "Incomplete",
+            "enum": [
                 "Incomplete",
-                "Complete",
-                "Inverified"
+                "Inverified",
```

Comment:
Should it be Unverified instead of Inverified?

Reply:
Corrected. Jason wants to leave this one for checking so I kept this field in each format part.